### PR TITLE
20201217

### PR DESCRIPTION
**Pull request by nomuun**
first commit

### CODE REVIEW

**Comment by nomuun** [comment]
d

**Review by nomuun — COMMENTED** [review]
d

File: README.md [hunk 1]
```diff
@@ -288,10 +288,7 @@ not just when a specific online course is in session.
 
 ## Choose a Programming Language
 
-You'll need to choose a programming language for the coding interviews you do,
```
Comment (andrzejsydor) [inline README.md:291]: What is the change about?

**Comment by nomuun** [comment]
No change

**Pull request closed without merging**